### PR TITLE
Bump Prow components to pick up support for multiple build clusters.

### PR DESCRIPTION
**Pull request by cjwagner**
for #6053 
/cc @BenTheElder

### CODE REVIEW

File: prow/Makefile [hunk 1]
```diff
@@ -20,23 +20,23 @@ ALPINE_VERSION           ?= 0.1
...
 GIT_VERSION              ?= 0.1
 # HOOK_VERSION is the version of the hook image
-HOOK_VERSION             ?= 0.193
+HOOK_VERSION             ?= 0.194
 # SINKER_VERSION is the version of the sinker image
-SINKER_VERSION           ?= 0.26
+SINKER_VERSION           ?= 0.27
 # DECK_VERSION is the version of the deck image
-DECK_VERSION             ?= 0.79
+DECK_VERSION             ?= 0.80
```
Comment (BenTheElder) [inline prow/Makefile]: we don't need to bump deck again though this shouldn't hurt

**Comment by cjwagner** [comment]
Removed the deck bump.
/hold cancel

**Comment by cjwagner** [comment]
Travis CI passed, but didn't report to github: https://travis-ci.org/kubernetes/test-infra/builds/327897357

**Comment by BenTheElder** [comment]
@cjwagner Can we use /skip for that?

**Comment by cjwagner** [comment]
`/skip` should only allow us to skip contexts associated with jobs controlled by Prow and only if the job does not need to be run on the PR.

**Pull request merged**